backoffice/styles/globals.css

```diff
@@ -90,6 +90,29 @@ a {
   background-color: black;
 }
 
+.pagination-helper {
```

Comment:
We should move pagination css to Pagination.module.css.

Reply:
I've configured and added the Pagination.module.css file as you proposed.

<img width="427" alt="image" src="https://github.com/user-attachments/assets/a60537ac-fb6b-4907-a8c3-7c5d27810200" />

Reply:
Great, thanks.

Reply:
Thanks for addressing the feedback.